main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 	router.HandleFunc("/users", createUser).Methods("POST")
 	router.HandleFunc("/users/{id}", updateUser).Methods("PUT")
 	router.HandleFunc("/users/{id}", deleteUser).Methods("DELETE")
-
+	// nerd
```

Comment:
You probably meant not to include this, can you remove it?

Reply:
Thank you for noticing! It was a side request from the team, but I can remove it if needed.